browser/about_flags.cc

```diff
@@ -460,6 +468,17 @@ const flags_ui::FeatureEntry::Choice kBraveSkusEnvChoices[] = {
 #define BRAVE_SHIELDS_FEATURE_ENTRIES
 #endif
 
+#if BUILDFLAG(IS_ANDROID)
+#define BRAVE_VERTICAL_TABS_FEATURE_ENTRY
+#else
+#define BRAVE_VERTICAL_TABS_FEATURE_ENTRY \
+    {kBraveVerticalTabsFeatureInternalName,  \
+    flag_descriptions::kBraveVerticalTabsName, \
+    flag_descriptions::kBraveVerticalTabsDescription, \
+    kOsWin, \
```

Comment:
If you want to try this feature on other platforms, add your platform here.

Reply:
Should we add macOS by default too?

Reply:
Yes we can but currently MacOS seems to have critical issues including crash. My mac isn't delivered yet so I can't deal with it for now. What do you think? should I add mac here?

Reply:
Please add MacOS for testing.

Reply:
Good news! I got my Mac and fixed reported problems. I think we can ship this on Mac too! 

<img width="814" alt="image" src="https://user-images.githubusercontent.com/5474642/181729671-00adb093-b84c-4892-b06b-63824aeb9b10.png">